### PR TITLE
Includes/excludes help

### PR DESCRIPTION
**Pull request by jglick**
Not obvious that the default has to be `*`, not empty, for includes.

I also found a buglet fixed in https://github.com/jenkinsci/git-plugin/pull/391.

@reviewbybees esp. @abayer

### CODE REVIEW

**Comment by abayer** [comment]
:bee: - hooray for more intuitiveness. =)

**Comment by andresrc** [comment]
:bee:

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java [hunk 1]
```diff
@@ -445,6 +445,14 @@ public String getDisplayName() {
         }
 
         @Restricted(NoExternalUse.class)
+        public FormValidation doCheckIncludes(@QueryParameter String value) {
+            if (value.isEmpty()) {
+                return FormValidation.warning("Did you mean to use * to match all branches?");
```
Comment (recena) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: :ant: for do not support to l10n

Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: Nothing else in this class is internationalized yet either, but I can do it for this new code.

Comment (recena) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource.java]: If you prefer we can do it in a separated PR.

**Pull request merged**